model/registry/messaging.yaml

```diff
@@ -139,7 +139,7 @@ groups:
           body size should be used.
         examples: 1439
         tag: messaging-generic
-      - id: operation
```

Comment:
This needs to be kept in the registry but marked as deprecated, see https://github.com/open-telemetry/semantic-conventions/actions/runs/8612652057/job/23602291780.

Reply:
I added it to deprecated attributes.